### ISSUE TITLE
Can this run on a Pi 3?

### ISSUE THREAD

**Issue by sigg3net** [netoptimizer/IPTV-Analyzer#31]
I was just wondering whether I could run this on a Raspberry Pi 3 ?

I am currently testing IPTV stream by streaming an unencrypted UDP MPEG2 TS to vlc (outputting to file /dev/null) which works, but doesn't really give me the data I need. Would a Pi 3 be up for running IPTV-Analyzer?

Cheers!

**Comment by johnyboi003** [comment]
also keen on if this should run on rpi3 with raspbian jessie

kernel : 4.4.50-v7+

**Comment by netoptimizer** [comment]
I've never tried to run this on any Raspberry Pi... the reason is that I know that the Ethernet interface on these devices actually goes through the USB port, and all kinds of latency, scheduling and USB overhead issues can occur due to this.

You can try to use it, but be careful that the drops detected are not caused by the device itself...

**Comment by oijkn** [comment]
Yes I can confirm that it works on a Raspberry PI 3 but before you can compile the module you must apply this patch : [link](https://github.com/netoptimizer/IPTV-Analyzer/issues/22#issuecomment-223170605)
